build.gradle

```diff
@@ -425,18 +434,19 @@ android {
 
 dependencies {
   implementation fileTree(dir: 'libs', include: ['*.jar'])
-  implementation 'androidx.core:core:1.7.0'
-  implementation 'androidx.activity:activity:1.4.0'
-  implementation 'androidx.fragment:fragment:1.4.1'
-  compileOnly 'com.github.spotbugs:spotbugs-annotations:4.5.3'
-  coreLibraryDesugaring 'com.android.tools:desugar_jdk_libs:1.1.5'
+  implementation platform('org.jetbrains.kotlin:kotlin-bom:1.9.23')
```

Comment:
I was getting some conflicts between transitive dependencies in the `org.jetbrains.kotlin` org.  This bom does not actually pull in additional dependencies, but just helps Gradle to resolve the conflicts: https://github.com/JetBrains/kotlin-wrappers?tab=readme-ov-file#using-in-your-projects